config/crd/bases/kafka.nais.io_aivenapplications.yaml

```diff
@@ -0,0 +1,107 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.5
+  creationTimestamp: null
+  name: aivenapplications.kafka.nais.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
+    type: date
+  - JSONPath: .status.synchronizationState
+    name: State
+    type: string
+  - JSONPath: .status.secretName
+    name: Name of secret
+    priority: 10
+    type: string
+  - JSONPath: .spec.serviceUsers[*].credentialsExpirationTime
+    name: Credentials expire
+    priority: 20
+    type: string
+  group: kafka.nais.io
+  names:
```

Comment:
It is possible to add "shortnames" to CRDs, (like `cm` for `configmap`, `ds` for `daemonset` etc.). Should we add `aivenapp` as a shortname?